asctb-api/src/functions/api.functions.ts

```diff
@@ -8,6 +8,14 @@ function addBiomarker(rowHeader: any, s: any) {
     s.b_type = BM_TYPE.G;
   }
   if (rowHeader[0] === 'BProtein' || rowHeader[0] === 'BP') {
+    s.name = s.name.replace('Protein', '');
+    if (s.name.indexOf('+') > -1){
+      s.name = s.name.replace('+', '');
+      s.proteinPresence = true;
+    } else if (s.name.indexOf('-') > -1){
+      s.name = s.name.replace('-', '');
+      s.proteinPresence = false;
```

Comment:
Graph outputs (Graph, JSON-LD, and OWL) will need to encode edges according to this too (@emquardokus what is the ontology terms used for positive and negative associations with these proteins?). This can be done as a separate PR.

Reply:
@bherr2 As far as I've seen, I've not seen any ontology have CD4- or CD4+ etc to be able to point  to, if so, we would have already done this. The only attempts I've seen were from James Overton and from Alex Diehl (CL). It doesn't seem like these are "easily" useable in the way we would like ,which is to simply have unique IDs to directly state whether the protein marker by flow cytometry is + or -; I have looked since the beginning of coding of the ASCT+B reporter.  Discussions have been had since the earliest of times with no good conclusions by anyone really.  

I checked out PRO (protein ontology) and I do not see distinguishing ontology IDs to represent the state of + or - or other states like ++, which also occurs; the Overton paper below mentions the need for a standard in order to represent them and suggests a standard,but that's from 2019 so relatively new and doesn't seem like it's been tackled with a functional solution, just proposal. 

**We may need to remove protein markers from the Reporter until we can come up with an adequate resolution. Having it represented wrong is worse than not being there because people are using this to make decisions with impact on research.**

Here are the 2 papers and the FlowCL bioconductor repo.  
http://bioconductor.org/packages/release/bioc/html/flowCL.html Courtot, Mélanie, Justin Meskas, Alexander D. Diehl, Radina Droumeva, Raphael Gottardo, Adrin Jalali, Mohammad Jafar Taghiyar, et al. 2014. “FlowCL: Ontology-Based Cell Population Labelling in Flow Cytometry.” Bioinformatics 31 (8): 1337–39. https://doi.org/10.1093/bioinformatics/btu807.
and
Overton, James A, Randi Vita, Patrick Dunn, Julie G Burel, Syed Ahmad Chan Bukhari, Kei-Hoi Cheung, Steven H Kleinstein, Alexander D Diehl, and Bjoern Peters. 2019. “Reporting and Connecting Cell Type Names and Gating Definitions through Ontologies.” BMC Bioinformatics 20 (Suppl 5): 182–182. https://doi.org/10.1186/s12859-019-2725-5.

Reply:
Thanks @emquardokus ! Could you check out what spoke is doing with this? They have proteins, cell types, and interactions, but I'm not sure how to best read what the edges mean. Check out https://spoke.rbvi.ucsf.edu/docs/index.html 
Their network looks like this:
![image](https://user-images.githubusercontent.com/34098595/145414025-d9fde3b9-b19c-43a7-b636-7fb3062c00bc.png)

I can follow up with them if it's not quite obvious to you too

Reply:
@bherr2 Hi, took a look at SPOKE liink above.  They have a TON of stuff, but everyone is avoiding the "elephant in the room"... because it's hard. Flow cytometry is such an important clinical assay output (it's used in hospitals for assessing blood every time you go and get "CBC complete blood count, and many others) that it seems so strange it's not captured in a computationally useful way for our work, but I continue to search. So far, like I said earlier, I've seen "attempts", but nothing widely used.